csi/src/server.rs

```diff
@@ -45,6 +45,26 @@ mod identity;
 mod mount;
 mod node;
 
+#[macro_use]
+extern crate failure;
+#[derive(Debug, Fail)]
+pub enum CSIError {
+    #[fail(display = "IO error: {}", error)]
+    Iscsiadm { error: String },
```

Comment:
If it's an Iscsiadm error, is it correct to be displaying "IO error"?

Reply:
fixed https://github.com/openebs/Mayastor/pull/304/commits/a8208afe6683479a8f40af63527b01a761ba19ef